test/test_device_branch_constructors.jl

```diff
@@ -60,44 +59,270 @@ end
+            :Fp__TapTransformer,
+        )
+        transformer_variable = PSI.get_variable(
+            op_problem_m.internal.optimization_container,
+            :Fp__Transformer2W,
+        )
+        if model == DCPPowerModel
+            # TODO: Currently Broken, remove variable bounds in HVDCUnbounded
+            # @test check_variable_unbounded(op_problem_m, :Fp__HVDCLine)
+
+            @test check_variable_bounded(op_problem_m, :Fp__TapTransformer)
+            @test check_variable_bounded(op_problem_m, :Fp__TapTransformer)
+        end
+
+        @test solve!(op_problem_m) == RunStatus.SUCCESSFUL
+
+        @test check_flow_variable_values(
+            op_problem_m,
+            :Fp__HVDCLine,
+            "DCLine3",
+            limits_min,
+            limits_max,
+        )
+        @test check_flow_variable_values(
+            op_problem_m,
+            :Fp__TapTransformer,
+            "Trans3",
+            -rate_limit,
+            rate_limit,
+        )
+        @test check_flow_variable_values(
+            op_problem_m,
+            :Fp__Transformer2W,
+            "Trans4",
+            -rate_limit2w,
+            rate_limit2w,
+        )
+    end
+end
+
+@testset "AC Power Flow Models Monitored Line Flow Constraints and Static Unbounded" begin
+    ratelimit_constraint_names = [
+        :RateLimitFT__Transformer2W,
+        :RateLimitTF__Transformer2W,
+        :RateLimitFT__TapTransformer,
+        :RateLimitTF__TapTransformer,
+        :RateLimitFT__HVDCLine,
+        :RateLimitTF__HVDCLine,
+    ]
+
+    system = PSB.build_system(PSITestSystems, "c_sys14_dc")
+
+    hvdc_line = PSY.get_component(HVDCLine, system, "DCLine3")
+    limits_from = PSY.get_active_power_limits_from(hvdc_line)
+    limits_to = PSY.get_active_power_limits_to(hvdc_line)
+    limits_min = min(limits_from.min, limits_to.min)
+    limits_max = min(limits_from.max, limits_to.max)
+
+    tap_transformer = PSY.get_component(TapTransformer, system, "Trans3")
+    rate_limit = PSY.get_rate(tap_transformer)
+
+    transformer = PSY.get_component(Transformer2W, system, "Trans4")
+    rate_limit2w = PSY.get_rate(tap_transformer)
+
+    template = get_template_dispatch_with_network(ACPPowerModel)
+    op_problem_m = OperationsProblem(template, system; optimizer = ipopt_optimizer)
+    @test build!(op_problem_m; output_dir = mktempdir(cleanup = true)) ==
+          PSI.BuildStatus.BUILT
+    # TODO: use accessors to remove the use of Symbols Directly
+    qFT_line_variable =
+        PSI.get_variable(op_problem_m.internal.optimization_container, :FqTF__HVDCLine)
+    pFT_line_variable =
+        PSI.get_variable(op_problem_m.internal.optimization_container, :FpFT__HVDCLine)
+
+    qFT_tap_variable = PSI.get_variable(
+        op_problem_m.internal.optimization_container,
+        :FqFT__TapTransformer,
+    )
+    pFT_tap_variable = PSI.get_variable(
+        op_problem_m.internal.optimization_container,
```

Comment:
same here

Reply:
sorry, that's old code we don't need to grab variable anymore, I'll remove that